front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceChartV2.tsx

```diff
@@ -1,339 +1,119 @@
-    if (formattedTrainSimulation) {
-      enableInteractivityV2(
-        chart,
-        formattedTrainSimulation,
-        CHART_AXES.SPACE_SPEED,
-        false,
-        setChart,
-        simulationIsPlaying,
-        updateTimePosition,
-        timeScaleRange,
-        departureTime,
-        setSharedXScaleDomain,
-        additionalAxes
-      );
-    }
-  }, [chart, additionalAxes]);
-
-  /**
-   * redraw the graph when resized horizontally (window resized)
-   */
-  useEffect(() => {
-    const debounceResize = () => {
-      let debounceTimeoutId;
-      clearTimeout(debounceTimeoutId);
-      debounceTimeoutId = setTimeout(() => {
-        createChartAndTrain();
-      }, 15);
-    };
-    window.addEventListener('resize', debounceResize);
-    return () => {
-      window.removeEventListener('resize', debounceResize);
-    };
-  }, [chart, formattedTrainSimulation, localSettings, resetChart]);
-
   return (
     <Rnd
       default={{
         x: 0,
         y: 0,
         width: '100%',
-        height: `${chartHeight}px`,
+        height: `${heightOfSpeedSpaceChartContainer}px`,
       }}
-      minHeight={CHART_MIN_HEIGHT}
+      size={{
+        width: '100%',
+        height: `${heightOfSpeedSpaceChartContainer + SPEEDSPACECHART_PADDING_BOTTOM}px`,
+      }}
+      minHeight={SPEEDSPACECHART_MIN_HEIGHT}
       disableDragging
       enableResizing={{
         bottom: true,
       }}
       onResizeStart={() => {
-        setChartBaseHeight(chartHeight);
-        onSetChartBaseHeight(chartHeight);
+        setBaseHeightOfSpeedSpaceChart(heightOfSpeedSpaceChart);
       }}
       onResize={(_e, _dir, _refToElement, delta) => {
-        setChartHeight(chartBaseHeight + delta.height);
-        onSetChartBaseHeight(chartBaseHeight + delta.height);
+        setHeightOfSpeedSpaceChart(baseHeightOfSpeedSpaceChart + delta.height);
       }}
     >
       <div
-        id={`container-${CHART_ID}`}
-        className="speedspace-chart chart"
-        style={{ height: `${chartHeight}px` }}
+        id="container-SpeedSpaceChart"
+        className="chart"
+        style={{ height: `${heightOfSpeedSpaceChartContainer}px` }}
       >
-        <button
-          type="button"
-          className="showSettingsButton"
-          aria-label={t('toggleSpeedSpaceSettings')}
-          title={t('toggleSpeedSpaceSettings')}
-          onClick={() => setShowSettings(!showSettings)}
-        >
-          {showSettings ? <ChevronLeft /> : <ChevronRight />}
-        </button>
-        <div>
-          <SpeedSpaceSettings
-            electrificationRanges={pathProperties.electrifications}
-            showSettings={showSettings}
-            onSetSettings={onLocalSetSettings}
-            speedSpaceSettings={speedSpaceSettings}
-            trainRollingStock={trainRollingStock}
+        <span className="chart-title">{t('speedSpaceChart')}</span>
+        {containerWidth && (
+          <SpeedSpaceChart
+            width={containerWidth as number}
```

Comment:
What do you think of that ? 
```suggestion
          <SpeedSpaceChart
            width={containerWidth || SPEEDSPACECHART_MIN_HEIGHT}
```

Reply:
done